applications/Unity.GrantManager/src/Unity.GrantManager.Web/Pages/GrantApplications/Details.js

```diff
@@ -177,7 +177,6 @@ $(function () {
                 PubSub.publish('AssessmentComment_refresh', { review: selectedReviewDetails });
                 assessmentUserDetailsWidgetManager.refresh();
                 assessmentScoresWidgetManager.refresh();
-                updateSubtotal()
```

Comment:
Not sure why it removed this updateSubtotal() but it is not in the file anywhere